packages/foreman/foreman/foreman.spec

```diff
@@ -725,7 +728,17 @@ make -C locale all-mo
 
 #use Bundler_ext instead of Bundler
 mv Gemfile Gemfile.in
-cp config/database.yml.example config/database.yml
+echo "gem 'activerecord-nulldb-adapter'" >> Gemfile.in
+cat <<DATABASE > config/database.yml
+production:
+  adapter: nulldb
+development:
+  adapter: nulldb
+DATABASE
+cat <<DB_SCHEMA > db/schema.rb
+ActiveRecord::Schema.define(version: 1) do
+end
+DB_SCHEMA
```

Comment:
As you suggested, I think it'd be good if this lived in Foreman itself as `bundler.d/nulldb.rb` and `config/database.yml.nulldb`. Not sure about the schema.